litert/vendors/intel_openvino/dispatch/device_context.cc

```diff
@@ -0,0 +1,185 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "litert/vendors/intel_openvino/dispatch/device_context.h"
+#if __ANDROID__
+#include <android/hardware_buffer.h>
+#endif  // __ANDROID__
+
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+#include <iostream>
```

Comment:
Can this be removed? Let's limit header includes to what is needed for the prod use case.

Reply:
Done